lib/elixir/lib/list.ex

```diff
@@ -577,6 +577,21 @@ defmodule List do
   end
 
   @doc """
+  Returns true if `list` starts with `prefix`.
+
+  ### Examples
+
+      iex> List.starts_with?([1, 2, 3], [1, 2])
+      true
+      iex> List.starts_with?([:alpha], [])
+      true
+      iex> List.starts_with?([], [:alpha])
+      false
+  """
+  @spec starts_with?(list, list) :: boolean
+  def starts_with?(list, prefix), do: :lists.prefix(prefix, enum)
```

Comment:
Sorry for being a bit nitpicky :P, but could we use the `do ... end` style of definition here for consistency with the rest of the file? :bowtie: